meteor/client/ui/RundownView/RundownTiming/RundownTimingProvider.tsx

```diff
@@ -207,12 +207,11 @@ export const RundownTimingProvider = withTracker<
 				this.dispatchLREvent(calmedDownNow)
 			}
 
-			const syncedEventTimeNow = Math.floor(now / 1000) * 1000
```

Comment:
I believe `syncedEventTimeNow` makes other calculations rounded perfectly to the whole second represented by the synced event. Without it some values may start jumping unexpectedly e.g. you'll get timers that count like 00:01, 00:02, 00:02, 00:04

Reply:
Additionally I believe this made the synced timers tick as closely to the system clock as possible, which I don't think is the case anymore.

Reply:
Curiously, that sort of sequence of counting was what I was seeing before removing `syncedEventTimeNow`

Reply:
Was that happening before you made any changes attempting to fix this bug?